### PR TITLE
EVM support OpenGov

### LINKED ISSUE #1295: EVM support for OpenGov

### PR DESCRIPTION
**Pull request by boundless-forest**
Close #1295 

There is a well tested precompile in the Moonbeam, reuse it to enable OpenGov functionality in the EVM. Note that Darwinia has a different precompile address fot it.

### CODE REVIEW

File: runtime/pangoro/src/migration.rs [hunk 1]
```diff
@@ -41,7 +41,20 @@ impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 }
 
 fn migrate() -> frame_support::weights::Weight {
-	frame_support::weights::Weight::zero()
+	// substrate
+	use sp_core::H160;
+	use sp_std::str::FromStr;
+
+	const REVERT_BYTECODE: [u8; 5] = [0x60, 0x00, 0x60, 0x00, 0xFD];
+	const CONVICTION_VOTING_ADDRESS: &str = "0x0000000000000000000000000000000000000602";
```
Comment (AurevoirXavier) [inline runtime/pangoro/src/migration.rs:49]: Need some comments.

Comment (boundless-forest) [reply runtime/pangoro/src/migration.rs]: Done.

File: runtime/pangolin/src/migration.rs [hunk 2]
```diff
@@ -49,6 +53,15 @@ fn migrate() -> frame_support::weights::Weight {
 	migration::clear_storage_prefix(b"TechnicalMembership", b"Members", &[], None, None);
 	migration::clear_storage_prefix(b"TechnicalMembership", b"Prime", &[], None, None);
 
+	const REVERT_BYTECODE: [u8; 5] = [0x60, 0x00, 0x60, 0x00, 0xFD];
+	// CONVICTION_VOTING_ADDRESS equals to the addr(1538) in the pallet-evm runtime.
+	const CONVICTION_VOTING_ADDRESS: &str = "0x0000000000000000000000000000000000000602";
+	EVM::create_account(
```
Comment (AurevoirXavier) [inline runtime/pangolin/src/migration.rs:59]: Why do we need to create an account for this?

Comment (boundless-forest) [reply runtime/pangolin/src/migration.rs:59]: In addition to inserting to the `AccountCode`, the `create_account` function also follows the substrate account management rule to increase the corresponding account references, just like normal smart contract accounts.

Comment (AurevoirXavier) [reply runtime/pangolin/src/migration.rs:59]: Do we need to create an account for any new precompile added?

Comment (boundless-forest) [reply runtime/pangolin/src/migration.rs:59]: Yeah.

**Pull request merged**